### ISSUE TITLE
Directly parse from client

### ISSUE THREAD

**Issue by tarioch** [csingley/ibflex#8]
Is it somehow possible to directly parse the response from the api?

As far as I'm able to figure it out right now the client does only download and the parser requires a file.

I worked around it by manually doing the etree but it would be nice if that would somehow be possible. What I currently do is:

```python
response = client.download(token, queryId)

root = ET.fromstring(response)
if root.tag != "FlexQueryResponse":
    raise FlexParserError("Not a FlexQueryResponse")
statement = parser.parse_element(root)
assert isinstance(statement, Types.FlexQueryResponse)
```

**Comment by csingley** [comment]
Does `parser.parse()` not work?

```
response = client.download(token, queryId)
statement = parser.parse(response)
assert isinstance(statement, Types.FlexQueryResponse)
```

>

**Comment by tarioch** [comment]
No it was trying to open a file with a name of the content.

**Comment by csingley** [comment]
I always forget this part of the ElementTree API.  Thanks for reporting.
Should be fixed by [cd72ed4](https://github.com/csingley/ibflex/commit/cd72ed4d353db111dc3b769c1b95c43d5df3041d).

♫We won't go until we get some, we won't go until we get some♫
